packages/react-template-helper/react-template-helper.js

```diff
@@ -5,7 +5,9 @@ checkNpmVersions({
 }, 'react-template-helper');
 
 const React = require('react');
+const { createRoot } = require('react-dom/client')
```

Comment:
Won't this throw not found error on React < 18?

Reply:
Good spot, probably. 
Is there a good way to test the package locally with Meteor? Something similar to yarn link. 

Wouldn't feel comfortable merging anything without testing properly. I've only verified this is working on React 18.

Reply:
I think it should be possible to move it bellow inside the function where you check for the React version.

Reply:
True! That works. I've pushed that fix.